README.md

```diff
@@ -38,47 +37,11 @@ to your screen.
 
 ![screenshot](https://cloud.githubusercontent.com/assets/252962/23072106/73a0656c-f528-11e6-9fcd-3c900d1d47d3.jpg)
 
-# Standing on the Shoulders of Giants
-
-So we shown it is now possible to do 60fps web-based augmented reality on a phone.
-This is great for sure but how did we get here ? **By standing on the shoulders of giants!**
-It is thanks to the hard work from others, that we can today reach this mythic 60fps AR.
-So I would like to thanks :
-
-- **three.js** for being a great library to do 3d on the web.
-- **artoolkit**! years of development and experiences on doing augmented reality
-- **emscripten and asm.js**! thus we could compile artoolkit c into javascript
-- **chromium**! thanks for being so fast!
-
-Only thanks to all of them, I could do my part : Optimizing performance from 5fps on high-end
-phone, to 60fps on 2years old phone.
-
-After all this work done by a lot of people, we have a *web-based augmented reality solution fast enough for mobile*!
+# What means "Marker based"
```

Comment:
This probably should read 'What "Marker based" means'

Reply:
changed :)